__tests__/Car.test.js

```diff
@@ -0,0 +1,50 @@
+import { Car } from '../src/main.js'
+
+describe('자동차 이름 관련 테스트', () => {
+  let name = ''
+  const INVALID_NAME_ERROR_MESSAGE = '자동차 이름을 입력해 주세요.'
+
+  beforeEach(() => {
+    name = '자동차'
+  })
+
+  it('자동차는 이름을 상태로 가질 수 있다', () => {
+    const car = new Car(name)
+
+    expect(car.name).toBe(name)
+  })
+
+  it('이름을 입력하지 않으면 에러가 발생한다', () => {
+    const car = () => new Car()
+
+    expect(car).toThrow(INVALID_NAME_ERROR_MESSAGE)
+  })
+
+  it('공백만 입력하면 에러가 발생한다', () => {
+    const car = () => new Car('  ')
+
+    expect(car).toThrow(INVALID_NAME_ERROR_MESSAGE)
+  })
```

Comment:
```suggestion
  it('이름을 입력하지 않으면 에러가 발생한다', () => {
    expect(() => new Car()).toThrow(INVALID_NAME_ERROR_MESSAGE)
  })

  it('공백만 입력하면 에러가 발생한다', () => {
    expect(() => new Car('  ')).toThrow(INVALID_NAME_ERROR_MESSAGE)
  })
```

지금은 오히려 변수로 선언하는게 더 어색해보이네요..!
아니면 변수 이름을 createCar 로 한다거나!?

Reply:
통일성을 고려해서 다른 테스트들과 동일하게 생성자를 변수로 선언했었습니다.
다만 전체적으로 변수를 선언만 해두고 사용하는 경우가 없어서, 선언 없이 바로 사용하는 방식으로 수정해보겠습니다!